projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.html

```diff
@@ -1,38 +1,46 @@
-<div class="search-file-name">
-  <span
-    tabindex="0"
-    role="link"
-    *ngIf="isFile"
-    (click)="showPreview($event)"
-    (keyup.enter)="showPreview($event)"
-    class="aca-link aca-crop-text"
-    [title]="nameStripped"
-    [innerHTML]="name$ | async"
-  ></span>
-  <span
-    tabindex="0"
-    role="link"
-    *ngIf="!isFile"
-    (click)="navigate($event)"
-    (keyup.enter)="navigate($event)"
-    class="bold aca-link aca-crop-text"
-    [title]="nameStripped"
-    [innerHTML]="name$ | async"
-  ></span>
-  <span
-    data-automation-id="search-results-entry-title"
-    class="aca-crop-text"
-    [title]="titleStripped"
-    [innerHTML]="title$ | async"
-  ></span>
+<div class="aca-search-results-text">
+  <div class="search-file-name">
+    <span tabindex="0"
+          role="link"
+          *ngIf="isFile"
+          (click)="showPreview($event)"
+          (keyup.enter)="showPreview($event)"
+          class="aca-link aca-crop-text"
+          [title]="nameStripped"
+          [innerHTML]="name$ | async"
+    ></span>
+    <span tabindex="0"
+          role="link"
+          *ngIf="!isFile"
+          (click)="navigate($event)"
+          (keyup.enter)="navigate($event)"
+          class="bold aca-link aca-crop-text"
+          [title]="nameStripped"
+          [innerHTML]="name$ | async"
+    ></span>
+    <span data-automation-id="search-results-entry-title"
+          class="aca-crop-text"
+          [title]="titleStripped"
+          [innerHTML]="title$ | async"
+    ></span>
+  </div>
+  <div data-automation-id="search-results-entry-description"
+       class="aca-crop-text"
+       [title]="descriptionStripped"
+       [innerHTML]="description$ | async"
+  ></div>
+  <div class="aca-result-location">
+    <aca-location-link [context]="context" [showLocation]="true"></aca-location-link>
+  </div>
+  <div class="aca-result-content aca-crop-text" [title]="contentStripped" [innerHTML]="content$ | async"></div>
 </div>
-<div
-  data-automation-id="search-results-entry-description"
-  class="aca-crop-text"
-  [title]="descriptionStripped"
-  [innerHTML]="description$ | async"
-></div>
-<div class="aca-result-location">
-  <aca-location-link [context]="context" [showLocation]="true"></aca-location-link>
+<div class="aca-name-column-badges">
+  <ng-container *ngFor="let badge of badges">
```

Comment:
There is no point in duplicating the code, if we need badges in new place let's move it to separate component on ACA level

Reply:
I've created a separate component for badges and moved all the code over there.